docs/source/release/v5.1.0/diffraction.rst

```diff
@@ -50,23 +61,28 @@ Improvements
 - The currently loaded calibration is now shown at the bottom of the GUI.
 - The location of the saved output files from the GUI is now shown in the messages log.
 - The save directory is now displayed in the status bar of the GUI.
-- The Inspect Background button of the Fitting tab is now only enabled when the selected run has had a background subtraction.
+- The Inspect Background button of the Fitting tab is now only enabled when the selected
+  run has had a background subtraction.
 
 Bugfixes
 ^^^^^^^^
 - The Engineering diffraction gui no longer goes behind the workbench window when a plot is clicked on.
-- A focussed run cannot be loeaded with the same x unit more than once
+- A focused run cannot be loaded with the same x unit more than once
 
 Single Crystal Diffraction
 --------------------------
 Improvements
 ^^^^^^^^^^^^
-- :ref:`CombinePeaksWorkspaces <algm-CombinePeaksWorkspaces>` now combines the modulation vectors present in the two workspaces, provided the total number of vectors is less than 3.
-- New algorithm :ref:`FindGoniometerFromUB <algm-FindGoniometerFromUB-v1>` for making UBs for runs at different goniometer angles share common indexing and determine the goniometer axis and rotation required to match UBs to a reference.
+- The :ref:`CombinePeaksWorkspaces <algm-CombinePeaksWorkspaces>` algorithm now combines the modulation
+  vectors present in the two workspaces, provided the total number of vectors is less than 3.
+- :ref:`FindGoniometerFromUB <algm-FindGoniometerFromUB-v1>` is a new algorithm for making UBs for runs at
+  different goniometer angles share common indexing, and determine the goniometer axis and rotation
+  required to match UBs to a reference.
 - New instrument geometry for MaNDi instrument at SNS
-- New algorithm :ref:`AddAbsorptionWeightedPathLengths <algm-AddAbsorptionWeightedPathLengths-v1>` for calculating the absorption weighted path length for each peak in a peaks workspace. The absorption weighted path length is used downstream from Mantid in extinction correction calculations
-- Can now edit H,K,L in the table of a peaks workspace in workbench (now consistent with Mantid Plot)
+- :ref:`AddAbsorptionWeightedPathLengths <algm-AddAbsorptionWeightedPathLengths-v1>` is a new algorithm
+  for calculating the absorption weighted path length for each peak in a peaks workspace.
+- H,K,L can now be edited in the table of a peaks workspace in workbench (now consistent with Mantid Plot)
 - The peaks workspace table display now contains a column showing the value of the intensity/sigma for each peak.
-- SliceViewer can now correctly display non-orthogonal axes for output of :ref:`MDNorm <algm-MDNorm>`
+- SliceViewer can now correctly display non-orthogonal axes for output of the :ref:`MDNorm <algm-MDNorm>` algorithm.
```

Comment:
Please can you add in another bullet into this Single Crystal section for a PR that got missed:
 - Ellipsoid integration now supported in IntegratePeaksMD with option to fix orientation of ellipsoid to have one axis parallel to Q of peak